packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -263,7 +263,7 @@ class SelectableRegion extends StatefulWidget {
     required final VoidCallback onCopy,
     required final VoidCallback onSelectAll,
   }) {
-    final bool canCopy = selectionGeometry.hasSelection;
+    final bool canCopy = selectionGeometry.hasSelection && selectionGeometry.status == SelectionStatus.uncollapsed;
```

Comment:
```suggestion
    final bool canCopy = selectionGeometry.status == SelectionStatus.uncollapsed;
```
it guarantee to hasSelection if it is uncollapsed